gwctl/pkg/policymanager/manager.go

```diff
@@ -196,6 +201,36 @@ func (p PolicyCRD) IsClusterScoped() bool {
 	return p.crd.Spec.Scope == apiextensionsv1.ClusterScoped
 }
 
+func (p PolicyCRD) Spec() map[string]interface{} {
```

Comment:
These helper functions seem to be purely concerned with printing at the moment. Lets keep them in the printer package.